### PR TITLE
Account settings page

### PR DESCRIPTION
**Pull request by poudyalanil**

### CODE REVIEW

File: src/containers/Account/UpdateInfo/styles.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+
+    .update-button {
+      display: flex;
+      flex-direction: row-reverse;
+
+      .save-btn {
+        background-color: var(--color-primary);
+        font-size: var(--font-family-base);
```
Comment (ft9dipesh) [inline src/containers/Account/UpdateInfo/styles.scss]: should be var(--font-size-base).

File: src/containers/Account/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+import UpdateInfo from './UpdateInfo';
+import UpdatePassword from './UpdatePassword';
+import styles from './styles.scss';
+
+const Account = (props) => {
+    const [route, setRoute] = useState(true);
+    const handleChange = () => {
+        setRoute(!route);
+    };
```
Comment (ft9dipesh) [inline src/containers/Account/index.js]: useCallback

File: src/containers/Account/UpdateInfo/index.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            console.log(err);
+        }
+        const response = await updateInfo({ username: user.username, firstName, lastName, email: userData.email, organization: userData.org, role: userData.role});
+        if (response) {
+            const updatedUser = await Api.getUser();
+            dispatch(authActions.setUser(updatedUser));
+            setUserData({ ...userData, success: true });
+            Toast.show('Profile has been successfully updated !!', Toast.SUCCESS);
+
+        }
```
Comment (ft9dipesh) [inline src/containers/Account/UpdateInfo/index.js]: All asynchronous calls (await) should be inside try block.

File: src/components/UserNav/index.js [hunk 4]
```diff
@@ -20,14 +20,18 @@ const UserNav = props => {
...
+    };
 
     const handleLogOut = useCallback(() => {
         dispatch(logout());
         history.push('/');
     }, [dispatch, history]);
 
     const getInitial = useCallback(() => user?.firstName?.[0], [user]);
-
+
```
Comment (timilsinabishal) [inline src/components/UserNav/index.js:30]: remove the whitespace in here

File: src/components/UserNav/index.js [hunk 5]
```diff
@@ -20,14 +20,18 @@ const UserNav = props => {
     const history = useHistory();
     const dispatch = useDispatch();
     const {user} = useSelector(state => state.auth);
+    
+    const handleAccountRoute = () => {
+        history.push('/account');
+    };
```
Comment (ft9dipesh) [inline src/components/UserNav/index.js]: useCallback

File: .github/workflows/validate.yml [hunk 6]
```diff
@@ -54,7 +54,6 @@ jobs:
         with:
           eslint: true
           stylelint: true
-        auto_fix: true
```
Comment (timilsinabishal) [inline .github/workflows/validate.yml:57]: any reason to remove this?

File: src/components/UserNav/index.js [hunk 7]
```diff
@@ -20,14 +20,18 @@ const UserNav = props => {
     const history = useHistory();
     const dispatch = useDispatch();
     const {user} = useSelector(state => state.auth);
+    
+    const handleAccountRoute = () => {
```
Comment (timilsinabishal) [inline src/components/UserNav/index.js]: won't using Link be easier?

File: src/containers/Account/UpdatePassword/index.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    const dispatch = useDispatch();
+
+    const [password, setPassword] = useState({
+        first: '',
+        confirm: '',
+        error: '',
+    });
+    const [oldPassword, setOldPassword] = useState('');
+
+    const handlePasswordChange = (target) => {
```
Comment (ft9dipesh) [inline src/containers/Account/UpdatePassword/index.js]: useCallback

File: .github/workflows/validate.yml (hunk 6 above)
Comment (ft9dipesh) [reply .github/workflows/validate.yml:57]: Actually, this option was causing a problem in the workflow, so it's okay to remove it.

**Review by ft9dipesh — APPROVED** [review]

**Pull request merged**